changelog.txt

```diff
@@ -1,5 +1,9 @@
 *** WooCommerce Subscriptions Core Changelog ***
 
+= 2.0.0 - 2022-xx-xx =
+* Dev - Retrieving users subscriptions order has been updated to use the WooCommerce specific APIs in WC_Subscriptions_Order.
+* Dev - Subscriptions meta is now retrieved and saved using WooCommerce specific APIs.
```

Comment:
IMO these two entries convey broader changes than are actually incoming. 

> `* Subscriptions meta is now retrieved and saved using WooCommerce specific APIs`

For example, this sounds as though all subscription meta retrieval and saves are now done via wc apis. I didn't review or merge that PR but I don't think that's quite the case yet is it. 

From what I can tell the following PRs have been merged since 1.9.0

- https://github.com/Automattic/woocommerce-subscriptions-core/pull/159
- https://github.com/Automattic/woocommerce-subscriptions-core/pull/161
- https://github.com/Automattic/woocommerce-subscriptions-core/pull/162
- https://github.com/Automattic/woocommerce-subscriptions-core/pull/168
- ~https://github.com/Automattic/woocommerce-subscriptions-core/pull/165~ 

The last one is completely dev-env related so can be ignored but the others are relevant. What I noticed is that some of these PRs are multifaceted and should really be broken into multiple entries.  

Here's my suggestion for the changelog. I think you can keep the first entry ('Retrieving users subscriptions order...') but replace the second entry with the following

```suggestion
* Dev - Deprecate the WC_Subscriptions_Order::get_meta() function. Use wcs_get_objects_property( $order, $meta_key, "single", $default ) instead.
* Dev - Update the wcs_get_objects_property() function to prevent calls to get_post_meta() on objects that support calling the get_meta() function.
* Dev - Remove the get_post_meta() call from WCS_Post_Meta_Cache_Manager::maybe_update_for_post_change()
* Dev - Replace code using get_post_type( $order_id ) with WC Data Store get_order_type()
* Dev - Replace all cases of update_post_meta() where an Order ID is passed to use update_meta_data() instead. 
```